docker-compose-dev.yml

```diff
@@ -12,6 +10,7 @@ services:
     - ioi_static:/opt/translate/app/static/
     - ioi_media:/opt/translate/app/media/
     - ioi_logs:/opt/translate/app/logs/
+    - ./secrets/google_service_account.json:/opt/sa.json:ro
```

Comment:
Is there any reason for not mounting it on `/opt/translate/app/secrets`?

Reply:
Thanks, I updated